docs/database/readme.md

```diff
@@ -0,0 +1,159 @@
+# Schema
+
+```mermaid
+---
+title: Airneis database model
+---
+
+classDiagram
+
+Categories <|-- Products
+Carts <|-- Status
+Products <|-- Carts
+Users <|-- Addresses
+Users <|-- Carts
+Carts <|-- CartsProducts
+Products <|-- CartsProducts
+Products <|-- ProductImages
+Materials <|-- ProductsMaterials
+Products <|-- ProductsMaterials
+
+class Users {
+    id increments notNull
+    fullName text notNull
+    email text unique notNull
+    isAdmin boolean default false notNull
+    isActive boolean default false notNull
+}
+
+class Categories {
+    id increments notNull
+    name text notNull
+    thumbnailUrl text notNull
+    slug text unique notNull
+    orderOfDisplay number notNull
+}
+
+class Products {
+    id increments notNull
+    name text notNull
+    slug unique notNull
+    categoryId number notNull
+    price number notNull
+    description text notNull
+    thumbnailUrl text notNull
+    outOfStock boolean default false notNull
+    isHighlanderOfTheMoment boolean default false notNull
+}
+
+class ProductImages {
+    id increments notNull
+    productId number notNull
+    imageUrl text notNull
+}
+
+class Materials{
+    id increments notNull
+    name text notNull
+}
+
+class ProductsMaterials {
+    id increments notNull
+    productId number notNull
+    materialId number notNull
+}
+
+class Addresses {
+    id increments notNull
+    userId number notNull
+    address text notNull
+    address2 text
+    city text notNull
+    state text notNull
+    zipCode text notNull
+    country text notNull
+    firstName text notNull
+    lastName text notNull
+    phoneNumber text notNull
+    comment text
+}
+
+class Carts {
+    id increments notNull
+    userId number notNull
+    statusId number notNull
+    shippingAddressId number notNull
+    billingAddressId number notNull
+    finalized_at date notNull
+}
+
+class Status {
+    id increments notNull
+    name text notNull
+    isOrder boolean notNull
```

Comment:
I think we should name it `hasOrder`. We want to know if it has a current order or not, ortherwise you can put `isOrdering`


We need to make it more explicit for the order's payment status, a developer's natural instinct is know to if it's been paid.  We should make the knowledge to be understandable in quick minute

Reply:
I named it like this to know if the line with this status is still a cart or if this line is an order but if it's clearer for most of the team I will change it.